templates/page.html

```diff
@@ -275,12 +275,14 @@ <h3>Developers</h3>
         <div class="release-news">
           <img src="{{ '/static/img/release.png'|asseturl }}" />
           <div class="release-news-body">
-            <h3>REANA 0.7.3</h3>
-            <h6>24 March 2021</h6>
+            <h3>REANA 0.7.4</h3>
+            <h6>06 July 2021</h6>
             <p>
-              Adds reana-client validation of workflow input parameters and environment images. Improves cluster resilience in case of job failures. Fixes HTCondor and Slurm integration for complex job commands.
+              Adds wildcard patterns support for some of the reana-client commands.
+              Allows to specify a memory limit for Kubernetes compute backend jobs.
```

Comment:
The major news of the release is the user-configurable memory limit, so for the illustrative picture you could take some reana workflow yaml with `kubernetes_memory_limit`, e.g. for roofit.  That might be more illustrative...

I'll send more concrete suggestionns for the leading text later.

Reply:
```
Allows to specify memory limits for Kubernetes compute backend jobs. Improves workflow validation of input parameters and environment images.
```

I guess two items is enough to keep only the most important news...

Reply:
BTW the Yadage RooFit example is nice, but it may be a bit long... Two options: (1) we can underline in red or otherwise highlight (say line background) for the concrete line that is setting the limits (which is coming at the end and so is not very visible); or (2) we can show serial example which is shorter and hence more easily readable (and there we could for extra clarity highlight as well, actually).